openslides/motions/migrations/0003_motion_comments.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from ..config_variables import get_config_variables
+
+
+def change_motions_comments(apps, schema_editor):
+    """
+    Index the comments fields in the config. Changing from an array to a dict with the
+    ids as keys. CHange all motions from an array for comments to a dict with the comments
+    field id as key to link motion comments and comments fields.
+    """
+    # We get the model from the versioned app registry;
+    # if we directly import it, it will be the wrong version.
+    ConfigStore = apps.get_model('core', 'ConfigStore')
```

Comment:
@normanjaeckel Can you check this again? Now without using the ConfigHandler.

Reply:
Looks good, not tested.